rules/test_not_named_test.yml

```diff
@@ -0,0 +1,14 @@
+id: test_not_named_test
+message: Test function not named test_.
+severity: error
+language: Rust
+# Test functions (annotated with #[test] or #[tokio::test]) be named test_ etc.
```

Comment:
Maybe `must be named test_ etc.` instead.

Reply:
Fixed.